src/codeu/chat/DerbyStore.java

```diff
@@ -16,10 +16,13 @@
 import codeu.chat.util.Logger;
 import codeu.chat.util.Uuid;
 
+import codeu.chat.common.DerbyDatabaseInteractions;
+
 import java.io.File;
+import java.io.IOException;
 import java.sql.Connection;
 
-public class DerbyStore {
+public class DerbyStore implements DerbyDatabaseInteractions {
 	
 	private String driver = "org.apache.derby.jdbc.EmbeddedDriver";
 	private String protocol = "jdbc:derby:../test";
```

Comment:
Maybe a comment explaining exactly where the database will be? Eventually you might want to rename it to something other than "test" as well.